lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -80,7 +80,13 @@ DefConstructor('\documentstyle OptionalSemiverbatim SkipSpaces Semiverbatim []',
     # So, we'll just try for a .cls, punting to OmniBus if needed.
     # If we start wanting to read style files by default, we'll still need to handle this
     # specially, since class (or sty files pretending to be) cover so much more.
-    LoadClass($class, options => $options, after => Tokens(T_CS('\compat@loadpackages')));
+    if (FindFile($class, type => 'cls', notex => !LookupValue('INCLUDE_STYLES'))) {
+      LoadClass($class, options => $options,
+        after => Tokens(T_CS('\compat@loadpackages'))); }
+    else {
+      InputDefinitions('OmniBus', type => 'cls', noerror => 1, handleoptions => 1);
+      RequirePackage($class, options => $options,
```

Comment:
I'm wondering if this should have the ```as_class=>1``` option? I think it is sorta intended for this purpose.

Reply:
Sounds reasonable, added!